components/logged_in/logged_in.tsx

```diff
@@ -32,6 +33,7 @@ export type Props = {
     children?: React.ReactNode;
     mfaRequired: boolean;
     enableTimezone: boolean;
+    theme: Theme;
```

Comment:
this theme is no longer needed because the OnboardingTaskList was moved to the LoggedInRoute Component. Will send a new PR https://github.com/mattermost/mattermost-webapp/pull/11781 to remove this unnecessary change. Same goes for the Fragment on this same file on line 158.